src/nvim/version.c

```diff
@@ -139,17 +139,9 @@ static char *(features[]) = {
 #if defined(UNIX)
 
   // only Unix can have terminfo instead of termcap
-# ifdef TERMINFO
-  "+terminfo",
-# else // ifdef TERMINFO
   "-terminfo",
-# endif // ifdef TERMINFO
 #else   // unix always includes termcap support
-# ifdef HAVE_TGETENT
-  "+tgetent",
-# else  // ifdef HAVE_TGETENT
   "-tgetent",
```

Comment:
IIRC `-tgetent` & `-terminfo` You'd have to ask @justinmk to verify, but last time I checked we were going to simplify version output by just removing all these things which aren't actually compile time options.